### PR TITLE
Update Code Generation instruction and folder path

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Update code generation instruction on contributing docs. 
- Update paths to match the new Repo structure.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,6 +24,12 @@
...
 
 **Line Endings :** If working on windows OS ensure git is installed with `Checkout Windows-style, commit Unix-style` option or `core.autocrlf` set to *true* in git config. If working on Unix based Linux or MacOS ensure git is installed with `Checkout as-is, commit Unix-style` option or `core.autocrlf` set to *input* in git config
+
+**Code Generation :** Generated code is found within the generated folder in each service. Code generation can be done using one of the methods listed below. 
+- Run `generate.ps1` found within the particular service directory. e.g. `sdk\compute\Microsoft.Azure.Management.Compute\src\generate.ps1` To use this you must have appropriate powershell modules installed. You can find instructions for installing those modules [here](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)
+- Run `generate.cmd` found within the tools directory in the repo root. Use the appropriate parameters e.g. `./tools/generate.cmd cognitiveservices/data-plane/Face` for client library or `./tools/generate.cmd storage/resource-manager` for management library.
```
Comment (weshaggard) [inline CONTRIBUTING.md]: @erich-wang isn't this approach not suggested any longer? IIRC we are trying to delete the tools directory completely from the repo.

File: Documentation/code-generation-instructions.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ RP is the resource provider's directory under SDKs, eg.: Compute
 To generate the `SdkInfo*.cs` file, please run the `generate.ps1` script
 
 ## Code generation artifacts
-If code is generated using `generate.ps1`, information related to the code generation gets logged in a `.txt` file under `src\SDKs\_metadata` eg.: [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/_metadata/compute_resource-manager.txt)
+If code is generated using `generate.ps1`, information related to the code generation gets logged in a `.txt` file under `sdk\_metadata` eg.: [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/_metadata/compute_resource-manager.txt)
```
Comment (isra-fel) [inline Documentation/code-generation-instructions.md]: Might as well move the example file to `sdk\` and update the url so it makes more sense..

File: CONTRIBUTING.md (hunk 1 above)
Comment (isra-fel) [reply CONTRIBUTING.md]: Exactly

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (isra-fel) [reply CONTRIBUTING.md]: > @erich-wang isn't this approach not suggested any longer? IIRC we are trying to delete the tools directory completely from the repo.

Exactly

**Comment by pakrym** [comment]
This PR has been open for a while, can it be merged or closed?

**Comment by chidozieononiwu** [comment]
I'll close this because the issue is being handled by china team.

**Pull request closed without merging**